packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ## TBD Release
 
+- Fixed error message shown when creating a config file that already exists. [#3647](https://github.com/zowe/zowe-explorer-vscode/issues/3647)
```

Comment:
This should probably be moved under the Bug fixes section 😋 
But it can be done with a future (release?) PR 😋